### DISCUSSION TITLE
[Q&A] 为何使用 GPT-4 API 时会返回 GPT-3 响应？

### DISCUSSION THREAD

**Discussion by fkgiteee** [lobehub/lobehub#591] [General Question] [answered]
### 💻 系统环境 | Operating System

macOS

### 🌐 浏览器 | Browser

Chrome

### 🐛 问题描述 | Bug Description

在使用 GPT-4 API ，我遇到了一个问题。我在请求中明确指定使用 GPT-4 模型，返回的结果似乎是来自 GPT-3。例如，当我询问模型版本时，得到的回答是“我是 GPT-3 模型”。

### 🚦 期望结果 | Expected Behavior

预期 API 应返回明确表示其为 GPT-4 模型的答案。

### 📷 复现步骤 | Recurrence Steps

设置 API 请求，使用模型参数 gpt-4.0。
发送问题：“你是什么版本的 GPT？”
观察返回的答案。

**Comment by fkgiteee** [comment]
![image](https://github.com/lobehub/lobe-chat/assets/122834546/5173b0eb-a0a8-4a92-a9bc-7c6c3daded23)

**Comment by arvinxx** [comment, marked as answer]
这是常见问题了。gpt4并不知道自己是gpt4。

可以试试这个问题来测试 3.5和4： 「鲁迅为何暴打周树人」

之前的3.5 并无法判断鲁迅和周树人是一个人。但 4 知道
